config/initializers/sidekiq.rb

```diff
@@ -25,4 +25,6 @@
   config.redis = { url: ENV['REDIS_URL'] }
 end
 
-Sidekiq::Queue['reverse_geocoding'].limit = 1 if PHOTON_API_HOST == 'photon.komoot.io'
+if defined?(Rails::Server) || Sidekiq.server?
+  Sidekiq::Queue['reverse_geocoding'].limit = 1 if PHOTON_API_HOST == 'photon.komoot.io'
```

Comment:
Then I'd suggest it like this:

```ruby
if (defined?(Rails::Server) || Sidekiq.server?) && PHOTON_API_HOST == 'photon.komoot.io'
  Sidekiq::Queue['reverse_geocoding'].limit = 1
end
```

Reply:
You are completely right. I was so focused on the first condition that missed the natural way of expressing the full condition. Forcepushed.